apis/notificationapi/notificationapi.yaml

```diff
@@ -15,10 +19,15 @@ paths:
       tags:
       - Notification APIs
       summary: ' Notifiying Users'
-      description: |-
-        This API is associated with notifying a user on the Sunbird Platform.if request will contain mode attribute with value sms ,then user will be notify via sms.During sms notification you can pass list of phone number or list of phone number and userid both.
-           - The endpoint for **Notifying Users** is `/user/v1/notification/email`
-           - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+      description: "This API is associated with notifying a user on the Sunbird Platform.if\
```

Comment:
@sknirmalkar89 This API description is not clear at all. Please explain. Also:
1. please do not use forced line breaks to format the text.
2. Start a new sentence with a capital letter. 
3. Check spelling and grammar

Reply:
updated